cmd/root.go

```diff
@@ -0,0 +1,53 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	homedir "github.com/mitchellh/go-homedir"
```

Comment:
you can get the user's homedir using internal go libraries, kind of like I did here:

https://github.com/ultralist/ultralist/blob/master/ultralist/file_store.go#L85

Reply:
That's what comes with cobra by default. Reason is described in https://github.com/mitchellh/go-homedir: 

*Why not just use os/user? The built-in os/user package requires cgo on Darwin systems. This means that any Go code that uses that package cannot cross compile. But 99% of the time the use for os/user is just to retrieve the home directory, which we can do for the current user without cgo. This library does that, enabling cross-compilation.*

Reply:
@timorunge hmm, but it looks like that library might be out of date.  Take a look at this: https://github.com/mitchellh/go-homedir/issues/20

Reply:
Yeah, it's also pointing [`os.UserHomeDir()`](https://golang.org/pkg/os/#UserHomeDir) which is even more simplistic.